test/manipulation.test.js

```diff
@@ -800,8 +800,8 @@ describe('manipulation', function() {
               p.id.should.equal(created.id);
               p.should.not.have.property('_id');
               p.title.should.equal('b');
-              p.should.not.have.property(p.content);
-              p.should.not.have.property(p.comments);
+              should.equal(p.content, undefined);
```

Comment:
In my experience, this code provides assertion (test failure) messages that are not very helpful:

```js
should.equal(p.content, 'undefined');
```

We usually use the following code instead:

```js
p.should.have.property('content', undefined);
```

Is there any particular reason against using this approach in this pull request?